discovery-provider/src/__init__.py

```diff
@@ -138,11 +139,11 @@ def create_celery(test_config=None):
     global web3endpoint, web3, abi_values, eth_abi_values, eth_web3
 
     web3endpoint = helpers.get_web3_endpoint(shared_config)
-    web3 = Web3(HTTPProvider(web3endpoint))
+    web3 = Web3(MultiProvider(web3endpoint))
```

Comment:
This is really just a multiprovider of size 1?

Reply:
This could be comma separated.